scripts/aspects_migration.js

```diff
@@ -0,0 +1,57 @@
+'use strict'
+// utility for reading in the entire curriculum from a file path
+const fs = require('fs')
+const Insight = require('../lib/insight')
+const childProcess = require('child_process').execSync
```

Comment:
name this `execSync` as `childProcess` is the name of the module and it's really counterintuitive naming:
```
const { execSync } = require('childProcess')
```